modules/ROOT/pages/design-edit-mulitple-tabs.adoc

```diff
@@ -0,0 +1,40 @@
+= Edit in Multiple Tabs in the Text Editor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can have multiple files open at a time in API Designer's text editor.
+
+== Opening Files in New Tabs
+
+There are four ways that you can open a file in a new tab:
+
+* Open the context menu for a file in the file tree by clicking the three dots to its right, and then select *Open in new tab*.
+* Press the Command key (MacOS) or the Control key (Windows) and click the name of the file in the file tree.
++[NOTE]
+====
+Clicking a filename in the file tree without first pressing either of these keys replaces the file that is open in the current tab.
+====
+
+* Place your cursor in the name of a file in a file reference such as an `include` tag in RAML or a `$ref` keyword in OAS. Then, press the Command key (MacOS) or the Control key (Windows) and click the filename.
+* Click a filename in the *Location* column in the errors view at the bottom of the screen.
+
+
+
+== Navigating to Different Tabs
+
+The tab bar appears at the top of the editor only when at least two files are open.
+
+To switch to the tab to the left of the currently open one, follow either of these steps:
+* Press Option + Shift + Tab (MacOS) or Alt + Shift + Tab (Windows).
+* Open the menu on the tab bar by clicking the ellipsis and select *Previous Tab*.
+
+To switch to the tab to the right of the currently open one, follow either of these steps:
+* Press Option + Tab (MacOS) or Alt + Tab (Windows).
+* Open the menu on the tab bar by clicking the ellipsis and select *Next Tab*.
+
+== Closing Tabs
+
+To close a single tab, click the X icon that appears on it.
+
+To close all tabs but the currently open one, click the ellipsis on the right side of the tab bar and select *Close All Other Tabs*.
```

Comment:
ditto "ellipsis"
Also, we are inconsistent in using both "to the right of" and "on the right side of."